app/validators/claim/base_claim_validator.rb

```diff
@@ -349,4 +356,12 @@ def validate_retrial_start_and_end(start_attribute, end_attribute, inverse = fal
   def validate_too_far_in_past(start_attribute)
     validate_on_or_after(Settings.earliest_permitted_date, start_attribute, 'check_not_too_far_in_past')
   end
+
+  def has_higher_rate_mileage?
+    @record.expenses.map { |s| s.mileage_rate_id.eql?(2) }.any?
+  end
+
+  def increased_travel?
+    @record.expenses.map { |x| x.calculated_distance && (x.distance > x.calculated_distance) }.any?
```

Comment:
not a blocker - could use a `.find`  - automatically breaks when first `true` result found. 

This will return true or nil tho.
```
@record.expenses.find { |x| x.calculated_distance && (x.distance > x.calculated_distance) }
```

Reply:
i have not this tested either

Reply:
All tests pass with the change.  Also `Benchmark.measure` makes it (a fraction of a second) faster, so I'll swap this in.

Reply:
🥇

Reply:
Both significantly faster by removing `.map` and replacing with simple `.where` clauses